### ISSUE TITLE
No Improvement in Accuracy with higher batch size like 8 (Multi GPU)

### ISSUE THREAD

**Issue by sree3333** [milesial/Pytorch-UNet#436]
I m trying to training over a dataset of 7k image with a batch size of 8 and 3 GPU parallely but once I increase batch size from 4 to 8, accuracy takes a hit and it wont improve from the first epoch itself

**Comment by leosmaia21** [comment]
Me too, if i use a batch other than 1 the accuracy stays the same

**Comment by ynma-hanvo** [comment]
maybe problem with the dataset.  
batchsize of 1 to 16 should be good .
